### ISSUE TITLE
Symbols function definition is void: dim:rcirc

### ISSUE THREAD

**Issue by codygman** [syl20bnr/spacemacs#854]
After putting the following in my .spacemacs:

```
;; To use this contribution add it to your ~/.spacemacs
(setq-default dotspacemacs-configuration-layers '(rcirc))

;; In dotspacemacs/config
      (setq rcirc-server-alist
            '(("irc.freenode.net"
               :user "spacemacs_user"
               :port "1337"
               :password "le_passwd"
               :channels ("#emacs"))))

;; In dotspacemacs/init
(setq rcirc-authinfo-support t)

;; In ~/.authinfo.gpg
machine irc.freenode.net port nickserv user spacemacs_user password le_passwd
```

I get the error:
```
Error (use-package): Symbol's function definition is void: dim:rcirc
```

**Comment by codygman** [comment]
I found these entries for it if that helps at all:

```
find . -type f -exec grep -nH -e dim {} +
./packages.el:51:        (defun dim:auth-source-fetch-password (server)
./packages.el:64:        (defun dim:rcirc-server-alist-get-authinfo (server-alist)
./packages.el:70:                   (pass  (dim:auth-source-fetch-password server)))
./packages.el:78:        (defun dim:rcirc ()
./packages.el:107:              (dim:rcirc-server-alist-get-authinfo
./packages.el:112:      (dim:rcirc)
```

**Comment by Futile** [comment]
I get a similar problem with powerline:
```
An error occurred while initializing powerline (error: (void-variable left))
Found 1 orphan package(s) to delete...
--> deleting rcirc-color... [1/1]
```
And then:
```
Error (use-package): Symbol's value as variable is void: left
```
After that, powerline doesn't work, but the rest of spacemacs seems to be working fine.
Version is 0.100.0 (with the Doge).

**Comment by syl20bnr** [comment]
@Futile see #817

**Comment by syl20bnr** [comment]
@codygman oops I should have seen this in the code review, I'll fix this tonight.

**Comment by Futile** [comment]
@syl20bnr missed that, thanks.

**Comment by syl20bnr** [comment]
Should be fixed in master (0.100.1)
